### DISCUSSION TITLE
spinning circle in design mode with safari not chrome

### DISCUSSION THREAD

**Discussion by Bogey** [Budibase/budibase#2933] [Help] [answered]
For some weird reason Safari won't display the Screens and Layouts of the Design mode.
It only shows the spinning 'loading...' circle (see screen shot).

![2021-10-08 16 31 28](https://user-images.githubusercontent.com/866768/136576727-a62696bf-c115-47c2-b97d-f03faf951cc1.gif)
On Chrome, it works as usual.
I've disabled content blockers, emptied caches, cleared cookies and what not.
Unfortunately no working design mode on Safari.

It's Safari 15.0 on MacOS 11.6

Thx,
Eddie

**Comment by ndlang** [comment]
I am also having this issue in Safari.
Safari Version 15.0 (16612.1.29.41.4, 16612)

**Reply by ndlang** [reply]
Works in Firefox version: 93.0

**Reply by Bogey** [reply]
On my iPad it works with neither Safari nor Chrome (Chrome on iPadOS uses the same Safari engine).

**Comment by mjashanks** [comment]
Thanks for reporting @ndlang & @Bogey - I've logged this as an issue here: https://github.com/Budibase/budibase/issues/2957

We'll get this fixed asap.

**Reply by Bogey** [reply]
Issue #2957 is closed as a dupe of #2945 and #2945 is closed while the issue isn't solved.
I still have no design view in Safari.

**Reply by mjashanks** [reply]
Just checking - have you updated to the latest version (released today)? `v0.9.173`

**Reply by shogunpurple** [reply]
This one hasn’t been done yet - 
https://github.com/Budibase/budibase/issues/2945 is also still open

**Reply by Bogey** [reply]
> Just checking - have you updated to the latest version (released today)? `v0.9.173`

Yes, just after I received the newsletter. I cleared cache too, but no luck.
On Firefox and Chrome it runs, but no Safari.

> This one hasn’t been done yet -
> #2945 is also still open

My apologies @shogunpurple I misinterpreted the purple 'closed' for the topic being closed.

**Comment by Bogey** [comment, marked as answer]
It looks like it's been fixed 👍🏻
